java-dataloader-3.1.0 has an invalid Automatic-Module-Name manifest entry

## Environment Details
* Helidon Version: 3.1.1
* Helidon SE, Helidon MP
* JDK version: N/A
* OS: N/A
* Docker version (if applicable): N/A

----------

## Problem Description
java-dataloader is a transitive dependency

```
[INFO] +- io.helidon.graphql:helidon-graphql-server:jar:3.1.1:compile
[INFO] |  \- com.graphql-java:graphql-java:jar:17.4:compile
[INFO] |     +- com.graphql-java:java-dataloader:jar:3.1.0:compile
[INFO] |     \- org.antlr:antlr4-runtime:jar:4.9.2:runtime
```

Its manifest defines an invalid automatic module name

```
Caused by: java.lang.module.FindException: Automatic-Module-Name: com.graphql-java: Invalid module name: 'graphql-java' is not a Java identifier
	at java.base/jdk.internal.module.ModulePath.deriveModuleDescriptor(ModulePath.java:505)
	at java.base/jdk.internal.module.ModulePath.readJar(ModulePath.java:644)
```

[v3.2.0](https://github.com/graphql-java/java-dataloader/releases/tag/v3.2.0) fixes this issue. I'd recommend upgrading when ready.

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -56,8 +56,8 @@
         <version.lib.google-error-prone>2.3.3</version.lib.google-error-prone>
         <version.lib.google-protobuf>3.21.7</version.lib.google-protobuf>
         <version.lib.graalvm>22.3.0</version.lib.graalvm>
-        <version.lib.graphql-java>17.5</version.lib.graphql-java>
-        <version.lib.graphql-java.extended.scalars>17.1</version.lib.graphql-java.extended.scalars>
+        <version.lib.graphql-java>19.5</version.lib.graphql-java>
+        <version.lib.graphql-java.extended.scalars>19.1</version.lib.graphql-java.extended.scalars>
         <version.lib.gson>2.9.0</version.lib.gson>
         <version.lib.grpc>1.54.1</version.lib.grpc>
         <version.lib.guava>31.1-jre</version.lib.guava>

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -56,8 +56,8 @@
         <version.lib.google-error-prone>2.3.3</version.lib.google-error-prone>
         <version.lib.google-protobuf>3.21.7</version.lib.google-protobuf>
         <version.lib.graalvm>22.3.0</version.lib.graalvm>
-        <version.lib.graphql-java>19.5</version.lib.graphql-java>
-        <version.lib.graphql-java.extended.scalars>19.1</version.lib.graphql-java.extended.scalars>
+        <version.lib.graphql-java>18.5</version.lib.graphql-java>
+        <version.lib.graphql-java.extended.scalars>18.3</version.lib.graphql-java.extended.scalars>
         <version.lib.gson>2.9.0</version.lib.gson>
         <version.lib.grpc>1.54.1</version.lib.grpc>
         <version.lib.guava>31.1-jre</version.lib.guava>

--- a/microprofile/graphql/server/src/main/java/module-info.java
+++ b/microprofile/graphql/server/src/main/java/module-info.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2022 Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2023 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/microprofile/graphql/server/src/main/java/module-info.java
+++ b/microprofile/graphql/server/src/main/java/module-info.java
@@ -38,7 +38,7 @@ module io.helidon.microprofile.graphql.server {
     requires io.helidon.microprofile.server;
 
     requires com.graphqljava;
-    requires graphql.java.extended.scalars;
+    requires com.graphqljava.extendedscalars;
     requires microprofile.graphql.api;
     requires microprofile.config.api;
 

